OSDOCS#14993: Adding monitoring cert-manager metrics section (#95058)
openshift/openshift-docs: security/cert_manager_operator/cert-manager-monitoring.adoc

toc::[]

> You can expose controller metrics for the {cert-manager-operator} in the format provided by the Prometheus Operator. You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.

Reviewer: ```suggestion
By default, the cert-manager emit metrics for all three of its core components: controller, cainjector, and webhook. You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.
```

// Enabling user workload monitoring for the cert-manager operand
include::modules/cert-manager-enable-user-workload-monitor.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]

Reviewer: ```suggestion
* xref:../../observability/monitoring/configuring-user-workload-monitoring/configuring-metrics-uwm.adoc#setting-up-metrics-collection-for-user-defined-projects_configuring-metrics-uwm[Setting up metrics collection for user-defined projects]
```

Author: changed as suggested

// Metrics scraping for cert-manager operands by using a ServiceMonitor
include::modules/cert-manager-enable-metrics.adoc[leveloffset=+1]

> 

Reviewer: ```suggestion
[role="_additional-resources"]
.Additional resources

* xref:../../observability/monitoring/configuring-user-workload-monitoring/preparing-to-configure-the-monitoring-stack-uwm.adoc#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Configuring user workload monitoring]

```

Author: @lunarwhite This section is added after the "Enabling user workload monitoring" as mentioned in gihub reference file . Should I add this again below "Configuring metrics scraping for cert-manager operands by using a ServiceMonitor" as suggested here?

Reviewer: For "enable-user-workload-monitor", reference would be "Setting up metrics collection for user-defined projects"

For "cert-manager-enable-metrics", reference would be "Configuring user workload monitoring"
